ansible/roles/test/tasks/fdb.yml

```diff
@@ -153,6 +154,9 @@
     command: sonic-clear fdb all
     when: sdk_fdb_count.stdout == show_mac_output.stdout
 
+  - fail: msg="No FDB is learned, something wrong with the switch"
+    when: sdk_fdb_count.stdout | int == 0 or show_mac_output.stdout | int == 0
```

Comment:
Just thinking, after fdb clear, the count would be 0 until it learns next right? I don't see a sleep in between. This I believe can break the test intermittently.

Reply:
Yes, config reloading would clear FDB. And then new FDB entries would be learned if the PTF script is still running. Currently the PTF script runs 300 seconds. After the PTF script is started, the config reload command is executed immediately. There is no need to sleep in between. If config reloading takes more than 300s to complete, the test will fail because of 0 FDB learned. In this case, the config reloading takes too long and is a problem that need to be fixed.